### PR TITLE
Use web component version of go ribbon.

### PR DESCRIPTION
**Pull request by rtaylorzfin**
Get the styling to match the legacy ribbon.
Match sizing.
Set height via margin.
Capitalize first 'a' in 'All annotations'.
include a label in API output (instead of null).
use "X" for indicating clicked cell.

### CODE REVIEW

**Review by rtaylorzfin — COMMENTED** [review]
Remove console.log

File: source/org/zfin/marker/presentation/GeneOntologyRibbonController.java [hunk 1]
```diff
@@ -27,6 +28,9 @@
     @Autowired
     private HttpServletRequest request;
 
+    @Autowired
```
Comment (rtaylorzfin) [inline source/org/zfin/marker/presentation/GeneOntologyRibbonController.java:31]: remove changes to this file.

File: home/css/_theme.scss [hunk 2]
```diff
@@ -30,7 +30,14 @@ $ribbon-strip-font-default: inherit;
 $ribbon-strip-tile-border-radius: 0;
 $ribbon-strip-tile-border-color: $border-color;
 $ribbon-strip-tile-border-color-hover: $border-color;
-$ribbon-strip-tile-tile-width: 20px;
+$ribbon-strip-tile-tile-width: 22px;
```
Comment (rtaylorzfin) [inline home/css/_theme.scss:33]: Put ribbon-specific styling in a ribbon.scss file

**Review by cmpich — APPROVED** [review]
Looks good.

**Pull request merged**